Cubical/Categories/Instances/Preorders/Monotone.agda

```diff
@@ -116,19 +116,19 @@ module _ {ℓ ℓ' : Level} where
   eqMon {X} {Y} f g p = isoFunInjective isoMonFunMonFun' f g (eqMon' _ _ p)
 
   -- isSet for Sigma
-  isSetSigma : {X Y : Preorder ℓ ℓ'} -> isSet (Sigma X Y)
-  isSetSigma {X} {Y} = isSetΣSndProp
-    (isSet→ (is-set (isPreorder (str Y))))
+  isSetSigma : {X Y : Preorder ℓ ℓ'} → isSet ⟨ Y ⟩ → isSet (Sigma X Y)
```

Comment:
Is this `Sigma` type used anywhere? Seems like it and all its associated functions can be deleted.

Reply:
Removed.